wp-content/themes/core/components/blocks/card_grid/Card_Grid_Controller.php

```diff
@@ -76,7 +76,7 @@ protected function defaults(): array {
 	protected function required(): array {
 		return [
 			self::CONTAINER_CLASSES => [ 'b-card-grid__container', 'l-container' ],
-			self::LOOP_CLASSES      => [ 'b-card-grid__loop' ],
+			self::LOOP_CLASSES      => [ 'b-card-grid__loop', 't-theme--light' ],
```

Comment:
Curious why the `t-theme--light` class is necessary, especially as a required class. Is this the default anyway?